### PR TITLE
First version

### PR DESCRIPTION
**Pull request by notFloran**

### CODE REVIEW

**Comment by notFloran** [comment]
@nclsHart @gignonje I think you can review

File: Enum/Endpoint.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Hexanet\OpenCloudBundle\Enum;
+
+class Endpoint
```
Comment (nclsHart) [inline Enum/Endpoint.php]: Maybe could you extends [SplEnum](http://php.net/manual/fr/class.splenum.php).

**Comment by gignonje** [comment]
LGTM

File: Enum/Endpoint.php (hunk 1 above)
Comment (notFloran) [inline Enum/Endpoint.php]: Not working

**Comment by nclsHart** [comment]
:+1:

**Pull request merged**